ReadersAndroidJetpack/app/src/main/res/xml/network_security_config.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <network-security-config>
     <domain-config cleartextTrafficPermitted="true">
-        <domain includeSubdomains="true">10.144.144.233</domain>
+        <domain includeSubdomains="true">10.144.24.74</domain>
```

Comment:
Can this be preprocessed so you’re not having to update the same value in two places? Or can a domain name be used to avoid the need to update it at all?